.g8/moneyPage/app/controllers/nonsipp/$if(directory.empty)$.$else$$directory$$endif$/$className$Controller.scala

```diff
@@ -16,6 +16,7 @@
 
 $! Generic imports !$
 $if(directory.empty)$
+
```

Comment:
Are these .g8 changes related to the addressLookup?

Reply:
yeah, to create some of the pages